tests/unit/ResultsListItem.spec.js

```diff
@@ -0,0 +1,49 @@
+import { mount } from '@vue/test-utils'
+import ResultsListItem from '@/components/ResultsListItem.vue'
+
+describe('ResultsListItem', () => {
+  it('renders ResultsListItem component', () => {
+    const wrapper = mount(ResultsListItem, {
+      provide: {
```

Comment:
It's refreshing to see that the syntax for `provide` and `propsData` doesn't stray too far from what we've experienced with React Testing Library.